templates/splash.html

```diff
@@ -408,11 +461,20 @@
     justify-content: space-between;
     align-items: flex-end;
   }
+  #now-playing {
+    padding: 10px 20px 10px 20px;
+    border-bottom-left-radius: 20px;  
+  }
   #qr-code {
     max-width: 25%;
+    border-top-right-radius: 20px; /* Rounded corner */
+    text-align: center; /* Center QR code */   
+    padding: 20px;
   }
   #up-next {
     max-width: 75%;
+    border-top-left-radius: 20px; /* Rounded corner */ 
+    padding: 10px 20px 10px 20px; 
```

Comment:
I had a look and I'm not crazy about these boxes. They cover up much more screen real estate than the text alone, most of it being blank space. If the goal is to make the text more readable, I feel that adding a black stroke effect to them would achieve the same thing and minimize how much of the video this is all covering up.  https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke

Reply:
On second thought, stroke is not great because it's an internal, not external stroke so it makes the text thinner. But an 8-axis shadow should work:
https://stackoverflow.com/a/47511171/2909171

Reply:
Hmm.... I really like the look, to me it makes the interface very polished looking, and on most my screens it's rare they cover up important words. 

I think what I might try to do is configure this so by default it's off, but ad a check box in the admin area so it can be turned on and off dynamically.

Reply:
I do like the general look, I think it's just the excessive blank space that bugs me. Maybe if the singer name goes on the same line as the song title, it would be cleaner. Then give it a max-width so it will wrap if necessary.

Reply:
btw, I don't think it's trivial to add admin settings checkboxes that will affect client side frontend. You'd probably have to dip into flask

Reply:
Another thought, maybe the project in general should allow css overrides for custom "skinning". There's no way any interface will make everyone happy.

Reply:
Ya, I'm definitely feeling that with the QR code area at the moment.
![image](https://github.com/vicwomg/pikaraoke/assets/72324013/237120ef-a45a-47d5-ad99-7cfc8f3eeac8)